pkg/synopsysctl/cmd_update.go

```diff
@@ -604,11 +604,11 @@ var updateBlackDuckAddEnvironCmd = &cobra.Command{
 	},
 }
 
-// updateBlackDuckAddRegistryCmd adds an Image Registry to a Black Duck instance
-var updateBlackDuckAddRegistryCmd = &cobra.Command{
-	Use:           "addregistry BLACK_DUCK_NAME REGISTRY",
-	Example:       "synopsysctl update blackduck addregistry <name> docker.io\nsynopsysctl update blackduck addregistry <name> docker.io -n <namespace>",
-	Short:         "Add an Image Registry to a Black Duck instance",
+// updateBlackDuckSetImageRegistryCmd adds an image to a Black Duck instance
+var updateBlackDuckSetImageRegistryCmd = &cobra.Command{
+	Use:           "setimage BLACK_DUCK_NAME (REGISTRY/IMAGE:TAG)",
+	Example:       "synopsysctl update blackduck setimage <name> docker.io/blackducksoftware/blackduck-cfssl:2019.6.0\nsynopsysctl update blackduck setimage <name> docker.io/blackducksoftware/blackduck-cfssl:2019.6.0 -n <namespace>",
+	Short:         "Update the registry location for an image used by a specific Black Duck instance",
```

Comment:
Since the command is setimage, sort should probably say "Set the ..." instead of "Update ...".  Sorry about the wording in my last comment